src/gui/QvisMainWindow.C

```diff
@@ -985,7 +984,7 @@ QvisMainWindow::SetDefaultSplitterSizes(int h)
 // ****************************************************************************
 // Method: QvisMainWindow::AddHelpMenu
 //
-// Purpose: 
+// Purpose:
```

Comment:
Many white-space changes here. Just curious why. Did you run some auto-tool over it?

Reply:
My editor is vi, is  use this command: %s/\s\+$//e 
to remove trailing whitespace from a file.
I've started doing it any time I am editing a file for commit.

Reply:
Ok, thats cool. But, it gets in the way of seeing the diffs essential to the work. So, I would propose doing it in some way that it can be filtered away from the diffs of the work the PR is submitted for. Maybe separate commits is sufficient? First commit white space changes, then commit the work. Again, I am NOT proposing any change here, now, in this PR. Just a change in practice to make it easier for PR reviewers